scss/inc/_test-layout.scss

```diff
@@ -93,12 +93,11 @@
     .test-sidebar-left {
         border-right: 1px $uiGeneralContentBorder solid;
 
-        .landmark-title--hidden {
+        .landmark-title-hidden {
```

Comment:
maybe we can use on generic class like .landmark-title-hidden to avoid dublication